tokenLists/fantom.mainnet.tokenlist.json

```diff
@@ -431,8 +422,7 @@
             "address": "0xFbc3c04845162F067A0B6F8934383E63899c3524",
             "_scan": "https://ftmscan.com/token/0xFbc3c04845162F067A0B6F8934383E63899c3524",
             "chainId": 250,
-            "decimals": 18,
-            "logoURI": "https://assets.spookyswap.finance/tokens/FOO.png"
+            "decimals": 18
```

Comment:
add logo

Reply:
The URL is no longer accessible and I couldn't find the logo URL in CoinMarketCap and ftmscan.com, so I removed it